Sources/Model/TransferMethod/HyperwalletTransferMethod.swift

```diff
@@ -133,10 +140,12 @@ public class HyperwalletTransferMethod: Codable {
     /// - bankAccount:   When the transfer method is Bank Account
     /// - bankCard:      When the transfer method is Bank Card
     /// - payPalAccount: When the transfer method is PayPal Account
+    /// - wireAccount:   When the transfer method is Wire Account
     public enum TransferMethodType: String {
         case bankAccount = "BANK_ACCOUNT"
         case bankCard = "BANK_CARD"
         case payPalAccount = "PAYPAL_ACCOUNT"
+        case wireAccount = "WIRE_ACCOUNT"
```

Comment:
Can we modify the build of HyperwalletBankAccount to this? 

```
  public init(transferMethodCountry: String,
                    transferMethodCurrency: String,
                    transferMethodProfileType: String,
                    transferMethodType: String) {
            storage[TransferMethodField.type.rawValue] = AnyCodable(value: transferMethodType)
            storage[TransferMethodField.transferMethodCountry.rawValue] = AnyCodable(value: transferMethodCountry)
            storage[TransferMethodField.transferMethodCurrency.rawValue] = AnyCodable(value: transferMethodCurrency)
            storage[TransferMethodField.profileType.rawValue] = AnyCodable(value: transferMethodProfileType)
        }
```

Reply:
Make sure all the tests pass and update all the usages of this builder

Reply:
@syyang-hw Requested changes provided. Tests pass, code coverage increased.